### PR TITLE
Create screen to search another user's MPE rooms

### PR DESCRIPTION
**Pull request by Devessier**
Closes https://github.com/AdonisEnProvence/MusicRoom/issues/355

### CODE REVIEW

File: packages/client/screens/UserMusicPlaylistEditorSearchScreen/index.tsx [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+            return <BlankScreen {...props} />;
+        }
+
+        const showLoadingIndicator = state.matches('Show loading indicator');
+        if (showLoadingIndicator === true) {
+            return <LoadingScreen {...props} />;
+        }
+
+        const userIsUnknown = state.matches('Unknown user');
+        if (userIsUnknown === true) {
```
Comment (prastoin) [inline packages/client/screens/UserMusicPlaylistEditorSearchScreen/index.tsx:346]: **praise:** loving the file readability

File: packages/client/screens/UserMusicPlaylistEditorSearchScreen/__tests__/UserMusicPlaylistEditorSearchScreen.test.tsx [hunk 2]
```diff
@@ -0,0 +1,762 @@
...
+                                        expect(
+                                            await withinOtherUserMpeRoomsScreen(
+                                                screen,
+                                            ).findByText(mpeRoom.roomName),
+                                        ).toBeTruthy();
+                                    }
+                                },
+                            },
+                        },
+                        'Loaded all MPE rooms': {
```
Comment (prastoin) [inline packages/client/screens/UserMusicPlaylistEditorSearchScreen/__tests__/UserMusicPlaylistEditorSearchScreen.test.tsx:271]: **question:** Unless I'm mistaken, pressing on a mpe card should also be tested isn't it ? We would expect to at least show the default mpeRoom view screen ( by default I mean room not found but would still attest the redirection )

**Comment by prastoin** [comment]
Great work !
One question to be solved before approving and mergin !

File: packages/client/screens/UserMusicPlaylistEditorSearchScreen/__tests__/UserMusicPlaylistEditorSearchScreen.test.tsx (hunk 2 above)
Comment (Devessier) [reply packages/client/screens/UserMusicPlaylistEditorSearchScreen/__tests__/UserMusicPlaylistEditorSearchScreen.test.tsx:271]: Indeed, it's an important part of the model, it should be tested 👍

**Review by prastoin — APPROVED** [review]
Great implementation !
Ready to be merged

**Pull request merged**